howto/install/deploy-bare-metal.md

```diff
@@ -0,0 +1,93 @@
+To deploy Anbox Cloud on a public cloud (such as AWS, Azure or Google) or using MAAS or OpenStack, see the instructions in [Deploy Anbox Cloud with Juju](https://discourse.ubuntu.com/t/deploy-anbox-cloud-with-juju/17744).
+
+Alternatively, you can follow the instructions in this document to use the [manual cloud provider](https://jaas.ai/docs/manual-cloud) that Juju offers. This method allows you to deploy Anbox Cloud with Juju on a set of SSH connected machines.
+
+## Prerequisites
+
+Before you start the installation, ensure that you have the required prerequisites:
+
+* At least three Ubuntu 18.04 LTS or 20.04 LTS machines. See [Minimum hardware](https://discourse.ubuntu.com/t/requirements/17734#minimum-hardware) for details and recommendations.
+* Your *Ubuntu Advantage for **Applications*** token. If you don't have one yet, [speak to your Canonical representative](https://anbox-cloud.io/contact-us). If you already have a UA Applications token, sign in on https://ubuntu.com/advantage to retrieve it.
+  [note type="caution" status="Warning"]The *Ubuntu Advantage for **Infrastructure*** token that every user gets for free for personal use does **NOT** work and will result in a failed deployment. You must purchase a *Ubuntu Advantage for **Applications*** subscription by [contacting Canonical](https://anbox-cloud.io/contact-us).[/note]
+
+## Install Juju on one machine
+
+Juju is a tool for deploying, configuring and operating complex software on public or private clouds.
+
+To install Juju 2.9, enter the following command:
+
+    sudo snap install --channel=2.9/stable juju
+
+See [Juju version](https://discourse.ubuntu.com/t/upgrading-from-previous-versions/17750#juju-version) for information about which Juju version is required for your version of Anbox Cloud.
+
+## Add a controller and model
+
+The Juju controller is used to manage the software deployed through Juju, from deployment to upgrades to day-two operations. One Juju controller can manage multiple projects or workspaces, which in Juju are known as *models*.
+
+You should dedicate one machine as the Juju controller. Run the following command to bootstrap the controller onto that machine:
+
+    juju bootstrap manual/<user>@<controller IP address> anbox-cloud
+
+Juju will connect to the machine via SSH as the specified user and install all necessary requirements.
+
+When the controller is set up, create a model to hold the Anbox Cloud deployment:
+
+    juju add-model main
+
+## Add all machines
+
+Before starting the deployment, you must add all machines to Juju. See [Minimum hardware](https://discourse.ubuntu.com/t/requirements/17734#minimum-hardware) for the list of machines that you need.
+
+When adding the machines, start with the machine that you want to host the management layer of Anbox Cloud. Then add all LXD worker nodes. Run the following command for each machine:
```

Comment:
Do we also need to add the machine that runs the controller?

Reply:
No, it's automatically added when you run the `juju bootstrap` command